src/server/pps/server/api_server.go

```diff
@@ -2158,25 +2079,6 @@ func (a *apiServer) CreatePipelineInTransaction(
 			return err
 		}
 
-		if !request.Reprocess {
```

Comment:
So `Reprocess` works by changing the pipeline's salt, so each datum gets a new ID.  When running the pipeline, it will make sure it processes the correct set of datums, so we don't actually have to do any funny business with parent/child commits.